apps/cognito-functions/src/custom-message-handler.ts

```diff
@@ -1,21 +1,24 @@
 import * as t from 'io-ts';
 import { CustomMessageTriggerEvent } from 'aws-lambda';
+import mjml2html from 'mjml';
+import { minify } from 'html-minifier';
+import customMessage from './templates/custom-message';
 
 export const CustomMessageEnv = t.type({
   domain: t.string,
 });
 export type CustomMessageEnv = t.TypeOf<typeof CustomMessageEnv>;
 
-export const emailTemplate = (href: string) => `
-Ciao,
-<br />
-Grazie per aver creato un account sul DevPortal!
-Non ti resta che confermarci la validità di questa e-mail.
-<br />
-<a href="${href}">Verifica e-mail</a>
-<br />
-Il bottone non funziona? Puoi usare il seguente link: <a href="${href}">${href}</a>
-`;
+export const emailTemplate = (href: string) => {
+  const emailMessage = mjml2html(customMessage(href)).html;
+
+  return minify(emailMessage, {
+    collapseWhitespace: true,
+    minifyCSS: true,
+    caseSensitive: true,
+    removeEmptyAttributes: true,
+  });
+};
```

Comment:
What about move this function into `apps/cognito-functions/src/templates/confirmation-email.ts` and rename it to `makeConfirmationEmail`?